aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,328 @@
+
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Models/TodoItem.cs
+  :language: c#
+  :emphasize-lines: 10
+
+
+Describing Response Types
+'''''''''''''''''''''''''
+Consuming developers are probably most concerned with what is returned; specifically response types, error codes (if not standard). These are handled in the xml comments and DataAnnotations.
+
+Take the ``Create()`` method for example, currently it returns only "201 Created" response by default. That is of course if the item is in fact created, or a "204 No Content" if no data is passed in the POST Body.  However, there is no documentation to know that or any other response. That can be fixed by adding the following piece of code.
+
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Controllers/TodoController.cs
+    :language: c#
+    :start-after: Create_Method
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 17,18,20,21
+
+.. image:: web-api-help-pages-using-swagger/_static/data-annotations-response-types.png
+
+Customizing the UI
+''''''''''''''''''
+The stock UI is very functional as well as presentable, however when building documentation pages for your API you want it to represent your brand or look and feel. 
+
+Accomplishing that task with the Swashbuckle components is simple but requires adding the resources to serve static files that would not normally be included in a Web API project and then building the folder structure to host those files.
+
+Add the ``"Microsoft.AspNetCore.StaticFiles": "1.0.0-*"`` NuGet package to the project either using one of the methods mentioned prior in this tutorial and then open the *Startup.cs* file to add the necessary code for the static file middleware.
+
+.. code-block:: c#
+    :emphasize-lines: 4
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+        {
+            app.UseStaticFiles();
+
+            app.UseMvcWithDefaultRoute();
+
+            // Enable middleware to serve generated Swagger as a JSON endpoint
+            app.UseSwagger();
+
+            // Enable middleware to serve swagger-ui assets (HTML, JS, CSS etc.)
+            app.UseSwaggerUi();
+            
+        }
+
+Acquire the core *index.html* file used for the Swagger UI page from the `Github repository <a href="https://github.com/swagger-api/swagger-ui/blob/master/src/main/html/index.html">`_ and put that in the ``wwwroot/swagger/ui`` folder and also create a new ``custom.css`` file in the same folder.
```

Comment:
Copy the  *index.html* file used for the Swagger UI  from the `Swagger Github repro <https://github.com/swagger-api/swagger-ui/blob/master/src/main/html/index.html>`__ into the *wwwroot/swagger/ui*. 
then show the pic (add  :scale: 100%)

Add a *custom.css* file to the *wwwroot/swagger/ui* folder.